src/input-format.md

```diff
@@ -59,6 +59,11 @@ This prevents an [inner attribute] at the start of a source file being removed.
 > [!NOTE]
```

Comment:
From @traviscross at https://github.com/rust-lang/rust/pull/148051#issuecomment-3567383545 related to `r[input.shebang.inner-attribute]` (just above this)

> Let's please document the behavior of frontmatter removal, including how it interacts with the documented shebang removal exception

There is no change to the shebang vs attribute detection.  I've been trying to think this through and so far I've not come up with a way to include frontmatter that doesn't feel artificial (saying what doesn't affect this is an open set).